### DISCUSSION TITLE
Primary key not supported for converter

### DISCUSSION THREAD

**Discussion by TKQhena** [supabase/supabase#19756] [Questions] [answered]
So I am a student working on a personal project using the supabase nuget package in C#. I followed instructions to the T but am currently stuck on an Exception that says Postgres.Attribute.PrimaryKey is not a supported dictionary key for converter. I honestly don't know what to do at this point if anyone has an answer to what is happening pls help. I have provided the full error screenshot below

![Capture](https://github.com/supabase/supabase/assets/104490020/982b02c6-d380-4409-a07c-1275a0ff8362)

**Comment by GaryAustin1** [comment, marked as answer]
You might look thru the repository for that client and see if there is any relevant issues.  I found this one, https://github.com/supabase-community/supabase-csharp/issues/72  but don't use that client so not really sure of the context.

**Reply by TKQhena** [reply]
Thanks' solution worked was just the same error as the one on the post. Following the same solution worked
